packages/desktop/components/menu-buttons/ProposalDetailsButton.svelte

```diff
@@ -88,11 +88,16 @@
 
 <div class="max-h-7 max-w-9 flex-none flex-initial overflow-visible relative">
     <MeatballMenuButton onClick={modal?.toggle} />
-    <Modal bind:this={modal} position={{ right: '0' }} classes="mt-1.5">
-        <div class="flex flex-col">
-            {#each buttons as button}
-                <MenuItem {...button} />
-            {/each}
-        </div>
-    </Modal>
+    <div class="fixed overflow-visible translate-x-9">
```

Comment:
we still have some overflow or z-index issues 🙈 
![image](https://github.com/iotaledger/firefly/assets/3624944/117c8276-5f95-4237-8efd-011601ab9089)